StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -76,7 +76,7 @@ private void HandleTypeDeclaration(SyntaxNodeAnalysisContext context)
             {
                 if (!XmlCommentHelper.HasDocumentation(declaration))
                 {
-                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, declaration.Identifier.GetLocation()));
+                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, declaration.Identifier.GetLocation(), ArrayEx.Empty<object>()));
```

Comment:
It's one that doesn't have `Array.Empty<T>()` (it doesn't compile even if you try to reference it directly).